magni_bringup/launch/core.launch

```diff
@@ -31,13 +31,15 @@
     -->
```

Comment:
It is important and I think it is true but must ask that  we only do this ONCE at motor node startup and we do NOT keep sending the controller board version at any other time later and certainly do not keep setting it every so often always.    True?

Reply:
Correct. This is only run at launch, and sets the ros parameter for the motor node. I believe the motor node only every sends this info once.

Reply:
Thank you, just a quick verification.